docs/source/en/api/pipelines/chroma.md

```diff
@@ -19,20 +19,20 @@ specific language governing permissions and limitations under the License.
 
 Chroma is a text to image generation model based on Flux.
 
-Original model checkpoints for Chroma can be found [here](https://huggingface.co/lodestones/Chroma).
+Original model checkpoints for Chroma can be found [here](https://huggingface.co/lodestones/Chroma1-HD).
```

Comment:
This is intentional. As if there's any other official compatible checkpoint released under https://huggingface.co/lodestones/, the users will likely notice it.